src/snapred/backend/dao/ingredients/NormalizationCalibrationIngredients.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel
+
+from snapred.backend.dao.ingredients.ReductionIngredients import ReductionIngredients
+from snapred.backend.dao.ingredients.SmoothDataExcludingPeaksIngredients import SmoothDataExcludingPeaksIngredients
+from snapred.backend.dao.state import InstrumentState
+from snapred.backend.dao.state.CalibrantSample import CalibrantSamples
+from snapred.backend.dao.state.FocusGroup import FocusGroup
+
+
+class NormalizationCalibrationIngredients(BaseModel):
+    """Class to hold the ingredients necessary for normalization calibration algo."""
+
+    reductionIngredients: ReductionIngredients
+    backgroundReductionIngredients: ReductionIngredients
+    calibrantSample: CalibrantSamples
+    focusGroup: FocusGroup
```

Comment:
Yep, just using FocusGroup for naming the workspaces, Malcolm wanted that for easier identification.

Reply:
Okay, this can be removed.  The grouping workspace will be loaded with FetchGroceries, which will automatically include the grouping scheme name in the workspace name